doc/source/guide/installation/linux.rst

```diff
@@ -5,11 +5,16 @@ Linux
 Overview
 --------
 
+.. warning::
```

Comment:
oh, maybe I just deleted it completely in favour of anaconda. This seems like something I might have done...